src/components/BrowserCell/BrowserCell.scss

```diff
@@ -77,4 +77,9 @@
 
 .readonly {
   color: #04263bd1;
-}
\ No newline at end of file
+}
+
+.action {
```

Comment:
Are you sure this doesn't have any effect on existing component styles?

Reply:
yes as we are only using the css with it's js name in the js file.